storage/app/core/minio.py

```diff
@@ -6,29 +6,36 @@
 from starlette.status import HTTP_500_INTERNAL_SERVER_ERROR
 
 logger = logging.getLogger("attachments")
+_minio_client = None
 
-minio_client = Minio(MINIO_HOST_URL,
-                     access_key=MINIO_ACCESS_KEY,
-                     secret_key=MINIO_SECRET_KEY,
-                     secure=False)
 
-try:
-    minio_client.list_buckets()
-    logger.info(f'Successfully connected to MINIO server on {MINIO_HOST_URL}')
-except Exception as exc:
-    logger.error(
-        "Error connecting to MINIO server on {MINIO_HOST_URL}: " + str(exc))
+def get_minio_client():
+    if _minio_client is None:
```

Comment:
needs a `global _minio_client` definition here otherwise the function can't read the variable defined above